tedana/resources/references.bib

```diff
@@ -313,3 +313,13 @@ @misc{sochat2015ttoz
   url       = {https://doi.org/10.5281/zenodo.32508},
   year      = 2015
 }
+
+@Article{Anglada:2022,
```

Comment:
Should this be `anglada2022`?

Got the following near the end of a run
```
File "/Users/logan/local_code/tedana/tedana/reporting/html_report.py", line 41, in _cite2html
    first_author = bibliography.entries[key].persons["author"][0]
  File "/Users/logan/miniconda3/envs/py310/lib/python3.10/site-packages/pybtex/utils.py", line 163, in __getitem__
    return self._dict[key.lower()]
KeyError: 'anglada2022'
```

And I notice that all other files in the `references.bib` are in that lower format + no ':'.  Maybe the case is handled, but seems like the colon not?

Reply:
Hi @dowdlelt. Yes, that is anglada2022. I am currently working on finalising this PR and all these issues will be resolved soon.

Reply:
awesome news, looking forward to it!